go/libraries/doltcore/sqle/enginetest/dolt_queries_rebase.go

```diff
@@ -0,0 +1,702 @@
+			"call dolt_commit('-am', 'inserting row 1');",
+			"alter table t add column c1 varchar(100) NOT NULL;",
+			"call dolt_commit('-am', 'adding column c1');",
+			"alter table t modify column c1 varchar(100) comment 'foo';",
+			"call dolt_commit('-am', 'altering column c1');",
+		},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query:    "call dolt_rebase('-i', 'main');",
+				Expected: []sql.Row{{0, "interactive rebase started"}},
+			},
+			{
+				Query: "select * from dolt_rebase order by rebase_order ASC;",
+				Expected: []sql.Row{
+					{"1", "pick", doltCommit, "inserting row 1"},
+					{"2", "pick", doltCommit, "adding column c1"},
+					{"3", "pick", doltCommit, "altering column c1"},
+				},
+			},
+			{
+				Query: "update dolt_rebase set rebase_order=3.1 where rebase_order=2;",
+				Expected: []sql.Row{{gmstypes.OkResult{RowsAffected: uint64(1), Info: plan.UpdateInfo{
+					Matched: 1,
+					Updated: 1,
+				}}}},
+			},
+			{
+				// Encountering a conflict during a rebase returns an error and aborts the rebase
+				Query:       "call dolt_rebase('--continue');",
+				ExpectedErr: dprocedures.ErrRebaseConflict,
+			},
+			{
+				// The rebase state has been cleared after hitting a conflict
+				Query:          "call dolt_rebase('--continue');",
+				ExpectedErrStr: "no rebase in progress",
+			},
+			{
+				// We're back to the original branch
+				Query:    "select active_branch();",
+				Expected: []sql.Row{{"branch1"}},
+			},
+			{
+				// The schema conflicts table should be empty, since the rebase was aborted
+				Query:    "select * from dolt_schema_conflicts;",
+				Expected: []sql.Row{},
+			},
+		},
+	},
+	{
+		// Tests that the rebase plan can be changed in non-standard ways, such as adding new commits to the plan
+		// and completely removing commits from the plan. These changes are also valid with Git.
+		Name: "dolt_rebase: non-standard plan changes",
+		SetUpScript: []string{
+			"create table t (pk int primary key);",
+			"call dolt_commit('-Am', 'creating table t');",
+			"call dolt_branch('branch1');",
+			"call dolt_branch('branch2');",
+
+			"insert into t values (0);",
+			"call dolt_commit('-am', 'inserting row 0');",
+
+			"call dolt_checkout('branch2');",
+			"insert into t values (999);",
+			"call dolt_commit('-am', 'inserting row 999');",
+
+			"call dolt_checkout('branch1');",
+			"insert into t values (1);",
+			"call dolt_commit('-am', 'inserting row 1');",
+			"insert into t values (2);",
+			"call dolt_commit('-am', 'inserting row 2');",
+			"insert into t values (3);",
+			"call dolt_commit('-am', 'inserting row 3');",
+		},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query:    "select active_branch();",
+				Expected: []sql.Row{{"branch1"}},
+			},
+			{
+				Query:    "select * from t;",
+				Expected: []sql.Row{{1}, {2}, {3}},
+			},
+			{
+				Query:    "call dolt_rebase('-i', 'main');",
+				Expected: []sql.Row{{0, "interactive rebase started"}},
```

Comment:
Seems like it might be useful to include more information in this message: what branch the rebase is using (if they get disconnected and need to reconnect), basic instructions on how to proceed with rebasing).

The other way to go would be to remove the information message altogether and let them read the docs to find out how to use the feature.

Reply:
I like having the human-readable status message that gives cues on what to do next. Updated this in 3ada516eaa532550bbc6592e112cb20963493ce7.